modules/nf-core/affy/justrma/tests/nextflow.config

```diff
@@ -0,0 +1,6 @@
+process {
+    withName: 'test_affy_justrma:AFFY_JUSTRMA' {
+        ext.args = '--sample_name_col name --build_annotation TRUE'
```

Comment:
Where does this get parsed? I don't see any further reference to it

Reply:
It was being read for the second test, but then not actually used as the name of the process had changed from pytest. Actually using it now.